src/utilities/date.ts

```diff
@@ -0,0 +1,11 @@
+export const generateDate = (): string => {
+  const date = new Date();
+  const year = date.getFullYear();
+  const month = date.getMonth() + 1 < 10 ? `0${date.getMonth()}` : date.getMonth();
+  const day = date.getDate();
+  const hours = date.getHours();
+  const minutes = date.getMinutes() < 10 ? `0${date.getMinutes()}` : date.getMinutes();
+  const seconds = date.getSeconds();
+
+  return `${year}-${month}-${day} ${hours}:${minutes}:${seconds}`;
+};
```

Comment:
It looks strange, surely there is no better way to implement it?

Reply:
Since we are using moment in our project you can simply do moment().format("YYYY-MM-D h:m:s")

Reply:
probably it's the best idea